src/main/java/uk/gov/hmcts/reform/sscs/config/jms/TribunalsHearingsJmsConfig.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.sscs.config.jms;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.jms.DefaultJmsListenerContainerFactoryConfigurer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.config.DefaultJmsListenerContainerFactory;
+import org.springframework.jms.config.JmsListenerContainerFactory;
+import org.springframework.jms.connection.CachingConnectionFactory;
+import org.springframework.jms.listener.DefaultMessageListenerContainer;
+
+import javax.jms.ConnectionFactory;
+import javax.jms.Session;
+
+@Slf4j
+@Configuration
+public class TribunalsHearingsJmsConfig {
+
+    @Value("${azure.service-bus.tribunals-to-hearings-api.connectionString}")
+    private String connectionString;
+
+    @Value("${azure.service-bus.tribunals-to-hearings-api.username}")
+    private String username;
+
+    @Value("${azure.service-bus.tribunals-to-hearings-api.password}")
+    private String password;
+
+    @Value("${azure.service-bus.tribunals-to-hearings-api.receiveTimeout}")
+    private Long receiveTimeout;
+
+    @Value("${azure.service-bus.tribunals-to-hearings-api.idleTimeout}")
+    private Long idleTimeout;
+
+    private final String connection = String.format("amqps://%1s?amqp.idleTimeout=%2d", connectionString, idleTimeout);
```

Comment:
I'd make the template a static final string then call the format in the factory method.